### PR TITLE
sfk 1.9.4.3

### PR DESCRIPTION
**Pull request by alebcay**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by alebcay** [comment]
```
checking whether the C++ compiler works... no
configure: error: in `/private/tmp/sfk-20190525-62482-1ix6aqx/sfk-1.9.4':
configure: error: C++ compiler cannot create executables
```

Not sure what's going on here (High Sierra and Mojave failure only).

File: Formula/sfk.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Sfk < Formula
   desc "Command-line tools collection"
   homepage "http://stahlworks.com/dev/swiss-file-knife.html"
-  url "https://downloads.sourceforge.net/project/swissfileknife/1-swissfileknife/1.9.2.0/sfk-1.9.2.tar.gz"
-  version "1.9.2.0"
-  sha256 "4537d033edfe6feb0bbafdcf157646a19951b8ef9056898718674d39a3c088d7"
+  url "https://downloads.sourceforge.net/project/swissfileknife/1-swissfileknife/1.9.4.3/sfk-1.9.4.tar.gz"
+  version "1.9.4.3"
```
Comment (zbeekman) [inline Formula/sfk.rb]: Does the version not get detected correctly without this? Can't this be removed?

**Comment by zbeekman** [comment]
isn't [this](https://github.com/Homebrew/homebrew-core/pull/40331/files#diff-6ca16fb03fb4e6bba185010d76563dc9R16) deprecated since xcode8?

i.e., 
```
ENV.libstdcxx
```

Will it build if we remove that? If not, smells like an issue needing to be reported upstream.

File: Formula/sfk.rb (hunk 1 above)
Comment (alebcay) [reply Formula/sfk.rb]: I'm pretty sure the version gets detected as `1.9.4`, which might make future updates more difficult (e.g. if `1.9.4.4` comes out).

**Comment by alebcay** [comment]
sfk 1.9.5.0 has been released. I'll go try my luck with that.

**Pull request closed without merging**